### PR TITLE
Refine Winery user guides

### PR DESCRIPTION
**Pull request by ana-silva**
Signed-off-by: Ana Cristina Franco da Silva <anacristina.francodasilva@gmail.com>

The following changes were done:
- content of 'modeling guide' was merged into the 'quickstart'
- a 'quicksetup' guide was added, which contains now the configuration information previously existent in the 'quickstart'
- FAQ was expanded
- documentation to Implementation Artifact generation was added  

- [x] Ensure that you followed http://eclipse.github.io/winery/dev/ToolChain#github---prepare-pull-request. Especially, we require **a single commit**
- [ ] Ensure that the commit message is [a good commit message](https://github.com/joelparkerhenderson/git_commit_message)
- [ ] Ensure to use auto format in **all** files
- [x] Ensure that you appear in `NOTICE` at Copyright Holders
- [ ] Tests created for changes
- [x] Documentation updated (if needed)
- [ ] Screenshots added (for UI changes)

### CODE REVIEW

**Comment by genie-winery** [comment]
Can one of the admins verify this patch?

File: docs/user/FAQ.md [hunk 1]
```diff
@@ -1,5 +1,55 @@
 # FAQ
 
-## Q: On Mac OS X, I neither can delete a node template nor a relationship template.
+## Q: What is TOSCA?
+
+A: Topology and Orchestration Specification for Cloud Applications (TOSCA) is an OASIS specification, which enables the modeling, provisioning and management of cloud applications. 
+For more detail, please click [here](http://eclipse.github.io/winery/tosca/).
```
Comment (lharzenetter) [inline docs/user/FAQ.md]: I would say `For more details see our [Notes on TOSCA](https://eclipse.github.io/winery/tosca/).`
Please always use `https`.

File: docs/user/FAQ.md [hunk 2]
```diff
@@ -1,5 +1,55 @@
...
+## Q: What is a CSAR?
+
+A: Cloud Service Archive (CSAR) is a packaging format defined by the TOSCA specification, which enables to bundle modeled TOSCA components in a self-contained manner. 
+In winery, you can model a service template and export it as a CSAR. 
+This CSAR can be loaded into the OpenTOSCA container in order to deploy your application. 
+
+## Q: How can I start the OpenTOSCA ecosystem?
+
+A: You can start the ecosystem by simply using Docker Compose or by using installation scripts. 
+Please refer to the [OpenTOSCA Getting Started guide](http://www.opentosca.org/sites/use_opentosca.html) for more details.
```
Comment (lharzenetter) [inline docs/user/FAQ.md]: Please always use `https`.

File: docs/user/FAQ.md [hunk 3]
```diff
@@ -1,5 +1,55 @@
...
+
+## Q: Is there an open repository for TOSCA types?
+
+A: Yes! We provide a [github repository](https://github.com/OpenTOSCA/tosca-definitions-public) compatible to Winery, which contains several service templates, node types, etc. 
+To use this repository with a Winery docker container, please refer to the corresponding [configuration instructions](https://github.com/OpenTOSCA/opentosca-docker#how-to-clone-a-private-tosca-definitions-repository-to-be-used-with-winery).
+
+## Q: Where can I find a quick start guide to model node types?
+
+A: You can find a Winery quick start guide about modeling node types [here](quickstart.md).
```
Comment (lharzenetter) [inline docs/user/FAQ.md]: Please do not use `here`, but rather the file name. E.g. `... node types in our [Quickstart](quickstart.md).`

File: docs/user/FAQ.md [hunk 4]
```diff
@@ -1,5 +1,55 @@
...
+A: Yes! We provide a [github repository](https://github.com/OpenTOSCA/tosca-definitions-public) compatible to Winery, which contains several service templates, node types, etc. 
+To use this repository with a Winery docker container, please refer to the corresponding [configuration instructions](https://github.com/OpenTOSCA/opentosca-docker#how-to-clone-a-private-tosca-definitions-repository-to-be-used-with-winery).
+
+## Q: Where can I find a quick start guide to model node types?
+
+A: You can find a Winery quick start guide about modeling node types [here](quickstart.md).  
+
+## Q: How can I export my modeled application as a CSAR?
+
+A: Select the Tab *Services Templates*.
```
Comment (lharzenetter) [inline docs/user/FAQ.md]: I guess `tab` should be lower case

File: docs/user/generateIA.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
+# Generate an Implementation Artifact for a Node Type Interface
+
+To specify what a node type should do, the user can define an *interface* and the *operations* provided by this interface. 
+Once the operations of a node type are defined, artifacts (e.g., shell scripts, .war files) implementing these operations need to be modeled. 
+For this, Winery provides an option to **automatically generate a stub java maven project** to build a **.war file** for a defined interface. 
+This can be done in the *Interface* tab of the node type view by pressing *Generate Implementation Artifact*. 
+A node type implementation will be automatically generated as well, if it does not exist yet.
+Once you get a confirmation that the artifact was generated, please press *Save*.
+
+ ![3-AddUbuntuNodeType](graphics/modeling/3-AddUbuntuNodeType.jpg)
```
Comment (lharzenetter) [inline docs/user/generateIA.md]: Maybe also highlight the *Generate Implementation Artifact* button in the screenshot

File: docs/user/generateIA.md [hunk 6]
```diff
@@ -0,0 +1,59 @@
...
+### 2) Manually
+ - Run: mvn clean package
+ - Locate the WAR file in the /target folder
+ - Open Winery in your browser, locate the artifact template representing this implementation artifact.
+ - Upload the WAR file to the artifact template in the tab *Files*
+
+
+## License
+
+Copyright (c) 2013-2017 Contributors to the Eclipse Foundation
```
Comment (lharzenetter) [inline docs/user/generateIA.md]: You created this file only this year, please adopt the year.

File: docs/user/FAQ.md [hunk 7]
```diff
@@ -1,5 +1,55 @@
...
 
 A: Select the node template (or the relationship template) and press <kbd>fn</kbd> + <kbd>backspace</kbd>.
+
+## Q: Where can I get more help?
+
+A: If you need support, contact us at *opentosca@iaas.uni-stuttgart.de*.
+
+## Q: How can I contribute to Winery?
+
+A: Please see the [contributing guide](https://github.com/eclipse/winery/blob/master/CONTRIBUTING.md).
```
Comment (lharzenetter) [inline docs/user/FAQ.md]: Please add the license here

File: docs/user/FAQ.md [hunk 8]
```diff
@@ -1,5 +1,69 @@
 # FAQ
 
-## Q: On Mac OS X, I neither can delete a node template nor a relationship template.
+## Q: What is TOSCA?
+
+A: Topology and Orchestration Specification for Cloud Applications (TOSCA) is an OASIS specification, which enables the modeling, provisioning and management of cloud applications.
```
Comment (saatkamp) [inline docs/user/FAQ.md]: Maybe add: The Topology and Orchestration Specification for Cloud Applications (TOSCA) is an OASIS standard to describe the deployment and management of applications in a portable manner. Based on standard-compliant TOSCA runtimes, such as the OpenTOSCA ecosystem, the deployment and management can be automated.

File: docs/user/FAQ.md [hunk 9]
```diff
@@ -1,5 +1,69 @@
...
 
-## Q: On Mac OS X, I neither can delete a node template nor a relationship template.
+## Q: What is TOSCA?
+
+A: Topology and Orchestration Specification for Cloud Applications (TOSCA) is an OASIS specification, which enables the modeling, provisioning and management of cloud applications. 
+For more details see our [Notes on TOSCA](https://eclipse.github.io/winery/tosca/).
+
+## Q: What is a CSAR?
+
+A: Cloud Service Archive (CSAR) is a packaging format defined by the TOSCA specification, which enables to bundle modeled TOSCA components in a self-contained manner.
```
Comment (saatkamp) [inline docs/user/FAQ.md]: Maybe add: Besides the TOSCA elements also the executable artifacts are packed.

**Pull request merged**